### DISCUSSION TITLE
Album IDs defined like weather location?

### DISCUSSION THREAD

**Discussion by aboell** [damongolding/immich-kiosk#160] [Ideas]
In the config.yaml file, is it possible to add label to represent an album, possibly in a similar way to how the weather location data is defined?  Since there's already an "album:" parameter, how about redefining it so it could add a label to the ID?  Then you could reference the labels for the albums in the URL in a similar way to how you reference the weather location.  

My use case I would love to provide frames for each of my children, but they live all over and they're not all detailed oriented.  Having them manually type in the long and unique album string could be a challenge.  This would simplify things so I could control it from the server side.

Example:
album:
  - name: My Wedding
     id: <album id abc>
  - name: Vacation
     id: <album id def>

**Comment by damongolding** [comment]
Not at the moment but it has been on my todo list for a little while. The name would have to be lowercase and no special characters though as it is parsed via the url.
